tests/puppeteer/pages/BO/advancedParameters/team/profiles.js

```diff
@@ -0,0 +1,138 @@
+require('module-alias/register');
+const BOBasePage = require('@pages/BO/BObasePage');
+
+module.exports = class Profiles extends BOBasePage {
+  constructor(page) {
+    super(page);
+
+    this.pageTitle = 'Profiles';
+
+    // Selectors
+    // Header links
+    this.addNewProfileLink = '#page-header-desc-configuration-add[title=\'Add new profile\']';
+    // List of profiles
+    this.profileGridPanel = '#profile_grid_panel';
+    this.profileGridTitle = `${this.profileGridPanel} h3.card-header-title`;
+    this.profilesListForm = '#profile_grid';
+    this.profilesListTableRow = `${this.profilesListForm} tbody tr:nth-child(%ROW)`;
+    this.profilesListTableColumn = `${this.profilesListTableRow} td.column-%COLUMN`;
+    this.profilesListTableToggleDropDown = `${this.profilesListTableColumn.replace('%COLUMN', 'actions')} 
+    a[data-toggle='dropdown']`;
+    this.profilesListTableDeleteLink = `${this.profilesListTableColumn.replace('%COLUMN', 'actions')} a[data-url]`;
+    this.profilesListTableEditLink = `${this.profilesListTableColumn.replace('%COLUMN', 'actions')} a[href*='edit']`;
```

Comment:
Same as page employees for column actions

Reply:
What's the problem here?

Reply:
it same suggestion as [here](https://github.com/PrestaShop/PrestaShop/pull/16289#discussion_r343035040)